test_data/makedata_suites/610_analyzers.js

```diff
@@ -0,0 +1,255 @@
+      // All items created must contain dbCount
+      // documentation link: https://www.arangodb.com/docs/3.10/analyzers.html
+      // model_cooking.bin : https://fasttext.cc/docs/en/supervised-tutorial.html
+
+      print(`making per database data ${dbCount}`);
+      function createAnalyzer(analyzerName, analyzerCreationQuery){
+        // creating analyzer
+        let text = createSafe(analyzerName,
+          function () {
+            return analyzerCreationQuery
+          }, function () {
+            if (a.analyzer(analyzerName) === null) {
+              throw new Error(`${analyzerName} analyzer creation failed!`);
+            }
+          });
+      }
+
+      //classifierSingle analyzer properties
+      //classifierSingle Analyzer which is capable of classifying tokens in the input text.
+      let classifierSingle = `classifierSingle_${dbCount}`;
+      let path = `${PWD}model_cooking.bin`;
+      let classifierSingleQuery = a.save(`${classifierSingle}`, "classification", { "model_location": `${path}` }, ["frequency", "norm", "position"]);
+      
+      //classifierDouble analyzer properties
+      //classifierDouble Analyzer which is capable of classifying tokens in the input text.
+      let classifierDouble = `classifierDouble_${dbCount}`;
+      let classifierDoubleQuery = a.save(`${classifierDouble}`, "classification", { "model_location": `${path}`, "top_k": 2 }, ["frequency", "norm", "position"]);      
+
+      //nearestNeighborsSingle analyzer properties
+      //nearestNeighborsSingle Analyzer capable of finding nearest neighbors of tokens in the input.
+      let nearestNeighborsSingle = `nearestNeighborsSingle_${dbCount}`;
+      let nearestNeighborsSingleQuery = a.save(`${nearestNeighborsSingle}`, "nearest_neighbors", { "model_location": `${path}` }, ["frequency", "norm", "position"]);      
+
+      //nearestNeighborsDouble analyzer properties
+      //nearestNeighborsDouble Analyzer capable of finding nearest neighbors of tokens in the input.
+      let nearestNeighborsDouble = `nearestNeighborsDouble_${dbCount}`;
+      let nearestNeighborsDoubleQuery = a.save(`${nearestNeighborsDouble}`, "nearest_neighbors", { "model_location": `${path}`, "top_k": 2 }, ["frequency", "norm", "position"]);      
+
+      //creating classifierSingle  analyzer
+      createAnalyzer(classifierSingle, classifierSingleQuery)
+      //creating classifierSingle  analyzer
+      createAnalyzer(classifierDouble, classifierDoubleQuery)
+      //creating nearestNeighborsSingle  analyzer
+      createAnalyzer(nearestNeighborsSingle, nearestNeighborsSingleQuery)
+      //creating nearestNeighborsDouble  analyzer
+      createAnalyzer(nearestNeighborsDouble, nearestNeighborsDoubleQuery)
+
+      return 0;
+    },
+    checkDataDB: function (options, isCluster, isEnterprise, database, dbCount, readOnly) {
+      print(`checking data ${dbCount}`);
+      progress(`checking data with ${dbCount}`);
```

Comment:
Both statements required?

Reply:
Updated and kept the first print statement only.